inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -115,8 +128,19 @@ def duplicated_doi_validator(form, field):
     if not doi:
         return
     if current_app.config.get('PRODUCTION_MODE'):
+        user_roles = _get_current_user_roles()
+
+        # First check in default collection
         inspirehep_duplicated_validator('doi:' + doi, 'DOI')
 
+        # And in Hal collection
+        try:
+            inspirehep_duplicated_validator('doi:' + doi, 'DOI', 'HAL Hidden')
```

Comment:
(Which is not something that you should fix in this PR, but something we should be aware of: we shouldn't have this repetition in the codebase)